### PR TITLE
add more math operators

### PR DESCRIPTION
**Pull request by mizzrawr**
Math.pow :penguin:

### CODE REVIEW

File: cindy/index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+//Q1. In one night, a movie theater sells tickets for 6450 dollars. Each ticket costs 15 dollars. How many tickets were sold?
+let dollars = 6450;
+let ticket = 15;
+let result = dollars / ticket;
+console.log(result + " tickets were sold");
+
+//Q2. Sylvia's income is 500 dollars per week. How much does Sylvia makes every year?
+let incomeWeek = 500;
+let weeks = 32;
```
Comment (ciamac-da) [inline cindy/index.js]: There is 52 weeks per year and not 32!!! So you have to multiply 500 to 52 instead!

**Review by ciamac-da — CHANGES_REQUESTED** [review]
Hey Cindy! Thanks for pushing! Please fix the second answer and push again!

File: cindy/index.js (hunk 1 above)
Comment (mizzrawr) [reply cindy/index.js]: omg that was my mistake :c fixed thanks for letting me know !

Comment (ciamac-da) [reply cindy/index.js]: You're welcome:)

**Review by ciamac-da — APPROVED** [review]
🥇

**Pull request merged**